cmd/spruce/main_test.go

```diff
@@ -250,6 +250,53 @@ map:
 			So(stderr, ShouldEqual, "")
 		})
 
+		Convey("Should not evaluate spruce logic appart from (( prune )) when --no-eval", func() {
```

Comment:
It's what it supposed to happen. If you specify --no-eval, then they prun operator and other operators aren't evaluated... 

not sure, what you expect here